src/codeu/chat/client/core/Context.java

```diff
@@ -47,4 +48,9 @@ public UserContext create(String name) {
     }
     return users;
   }
+  
+  public ServerInfo getInfo() {
+	View infoView = (View)view;
```

Comment:
Like I mentioned in the chat, consider exposing the `getInfo()` method on the `BasicView` interface so that there is no need for casting.